modules/ui/field.js

```diff
@@ -28,6 +29,8 @@ export function uiField(context, presetField, entityIDs, options) {
     var _state = '';
     var _tags = {};
 
+    options.source = field.source !== undefined ? field.source : true;
```

Comment:
If I understand correctly, `field.source` is undefined for most fields, but for the main Sources field, it’s set to `false`, correct?

Ideally, we’d have a way to set it to `true` for all the other fields over in the part of the code where we customize the preset data, which would be easier to find than this special case. But I see that we also have [a special case](https://github.com/OpenHistoricalMap/iD/blob/a5a46ae34e3eef0ba5b69e7f89d7834fc3c5073d/modules/ui/sections/preset_fields.js#L72-L79) elsewhere to force `start_date`, `end_date`, and `source` fields onto every preset, so maybe it isn’t so straightforward. (By the way, this is where we define the order of these fields; if we want `source` to come first, this is what would change.)

Since this is probably as good as it’ll get without a refactor of id-tagging-schema, let’s add a comment here to explain the reason for falling back to `true`.